certif/app/components/enrolled-candidates.hbs

```diff
@@ -0,0 +1,131 @@
+<div class="panel">
+  <div class="panel-header">
+    <div class="panel-header__title">
+      Liste des candidats ({{@certificationCandidates.length}})
+      {{#if this.isCandidateBeingAdded}}
+        <div class="panel-header__mandatory-warning">Les champs marqués de * sont obligatoires</div>
+      {{/if}}
+    </div>
+    {{#unless this.isScoSession}}
+      <div data-test-id="add-certification-candidate-staging__button" class="panel-header__action" {{on 'click' this.addCertificationCandidateInStaging}} >
+        <div class="certification-candidates-add-button__text">
+          Ajouter un candidat
+        </div>
+        <PixActionButton @icon='plus' />
```

Comment:
**Question :**  Niveau accessiblité, il est souvent conseillé que le bouton porte un texte est pas seulement une icone. afin qu'une personne naviguant au clavier ne tombe pas sur un bouton sans savoir ce qu'il fait. 

Je ne suis pas encore familier avec Ember le `{{on 'click' this.addCertificationCandidateInStaging}}` ne devrait-il pas être sur le button ?  dans un même souci d'accessibilité. Le focus risque de ne pas se porter sur la div mais seulement sur le button.